src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -535,6 +606,41 @@ class FileJobStoreTest(hidden.AbstractJobStoreTest):
     def _createJobStore(self, config=None):
         return FileJobStore(self.namePrefix, config=config)
 
+    @classmethod
+    def _getUrlForTestFile(cls, size=None):
+        fileName = 'testfile_%s' % uuid.uuid4()
+        dirPath = cls._externalStore()
+        localFilePath = dirPath + fileName
+        url = 'file://%s' % localFilePath
+
+        if size is None:
```

Comment:
There are a too many empty lines here for my taste. And despite an `else:` clause isn't exactly necessary because of the `return` in the `if` branch, I like to throw it in for clarity and symmetry.